src/iccpd/src/mlacp_link_handler.c

```diff
@@ -1333,40 +2361,47 @@ static void mlacp_fix_bridge_mac(struct CSM* csm)
 void mlacp_peer_conn_handler(struct CSM* csm)
 {
     struct LocalInterface *lif = NULL;
+    struct PeerInterface* peer_if;
     static int once_connected = 0;
     struct System* sys = NULL;
+    struct If_info * cif = NULL;
 
     if (!csm)
         return;
 
     if ((sys = system_get_instance()) == NULL)
+    {
+        ICCPD_LOG_ERR(__FUNCTION__, "Invalid system instance");
         return;
+    }
 
     if (csm->warm_reboot_disconn_time != 0)
     {
         /*If peer reconnected, reset peer disconnect time*/
         csm->warm_reboot_disconn_time = 0;
-        ICCPD_LOG_NOTICE(__FUNCTION__, "Peer warm reboot and reconnect, reset peer disconnect time!");
+        ICCPD_LOG_DEBUG(__FUNCTION__, "Peer warm reboot and reconnect, reset warm disconnect time!");
     }
 
     if (csm->peer_link_if)
     {
         set_peerlink_mlag_port_learn(csm->peer_link_if, 0);
     }
 
+    ICCPD_LOG_NOTICE("ICCP_FSM", "ICCP session up: warm reboot %s, role %s",
+        (sys->warmboot_start == WARM_REBOOT) ? "yes" : "no",
+        (csm->role_type == STP_ROLE_STANDBY) ? "standby" : "active");
+
     /*If peer connect again, don't flush FDB*/
     if (once_connected == 0)
     {
         once_connected = 1;
         mlacp_fix_bridge_mac(csm);
-        /*If warm reboot, don't flush FDB*/
-        if (sys->warmboot_start != WARM_REBOOT)
-            mlacp_clean_fdb();
+        // do not required to flush FDB
+        //if (sys->warmboot_start != WARM_REBOOT)
```

Comment:
Is this unwanted functionality?

Reply:
@prvattem: Yes, This flush is not required when ICCPd session is coming up. The unwanted flush is being commented.